db/migrate/20200317211000_drop_nullable_requirement_from_site_banners.rb

```diff
@@ -0,0 +1,6 @@
+class DropNullableRequirementFromSiteBanners < ActiveRecord::Migration[6.0]
+  def change
+    change_column_null :site_banners, :donation_amounts, true
```

Comment:
Have you checked the Site Banner code to see what happens when someone creates a new account / channel with a nil site banner? I think I remember that any new site banner creates with a tip set

Reply:
Yeah it works just fine.